crates/router/src/db/configs.rs

```diff
@@ -45,15 +45,15 @@ impl ConfigInterface for Store {
         &self,
         config: storage::ConfigNew,
     ) -> CustomResult<storage::Config, errors::StorageError> {
-        let conn = pg_connection_read(self).await?;
+        let conn = connection::pg_connection_write(self).await?;
         config.insert(&conn).await.map_err(Into::into).into_report()
     }
 
     async fn find_config_by_key(
         &self,
         key: &str,
     ) -> CustomResult<storage::Config, errors::StorageError> {
-        let conn = pg_connection_read(self).await?;
+        let conn = connection::pg_connection_read(self).await?;
```

Comment:
Config read also use write itself

Reply:
Why?

Reply:
In dashboard, will updating a config value for feature flags we need to check it correctly updated or not, we need to read from master and not replica

Reply:
Fixed